src/6_branch.js

```diff
@@ -983,23 +983,7 @@ Branch.prototype['link'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
  *
  * ___
  *
- * # Referral system rewarding functionality
- * In a standard referral system, you have 2 parties: the original user and the invitee. Our system
- * is flexible enough to handle rewards for all users for any actions. Here are a couple example
- * scenarios:
- * 1. Reward the original user for taking action (eg. inviting, purchasing, etc)
- * 2. Reward the invitee for installing the app from the original user's referral link
- * 3. Reward the original user when the invitee takes action (eg. give the original user credit when
- *     their the invitee buys something)
- *
- * These reward definitions are created on the dashboard, under the 'Reward Rules' section in the
- * 'Referrals' tab on the dashboard.
- *
- * Warning: For a referral program, you should not use unique awards for custom events and redeem
- * pre-identify call. This can allow users to cheat the system.
- *
- * ## Retrieve referrals list
- *
+ * ## Deepview
```

Comment:
no, this is right - since the readme is formed only from the comments, and the comment needs start with the function definition, i think we need to do this